scripts/helper/src/utils.ts

```diff
@@ -162,9 +162,7 @@ export const calcWeightPerSecond = (precision: number, price: number): number =>
   /// max_fee = (weight_per_second * weight)/WEIGHT_PER_SECOND/(10**precision) * price
   /// so weight_per_second = max_fee*WEIGHT_PER_SECOND*(10**precision)/weight/price
   const weight_per_second = (((max_fee * WEIGHT_PER_SECOND) / weight) * 10 ** precision) / price
-  /// to avoid price sharply increased later so that we charge too much
-  /// just add some soft limit here
-  return Math.min(1000 * WEIGHT_PER_SECOND, Math.floor(weight_per_second))
+  return Math.floor(weight_per_second)
```

Comment:
why do this？

Reply:
this script is for calculating `weight_per_second` and later used to register foreign asset. 

previously we add some soft limit to avoid price increased a lot and then we charge too much, but actually xcm fee we set here is low enough(0.02$ at most) so should not be a problem even asset price increased a lot. 

and we can change it without runtime upgrade anytime so just remove it to make code clean